rl_zoo3/plots/plot_train.py

```diff
@@ -58,11 +58,9 @@ def plot_train():
 
     for env in envs:
         dirs.extend(
-            [
-                os.path.join(log_path, folder)
-                for folder in os.listdir(log_path)
-                if (env in folder and os.path.isdir(os.path.join(log_path, folder)))
-            ]
+            entry.path
+            for entry in sorted(os.scandir(log_path), key=lambda e: e.stat().st_mtime)
+            if env in entry.name and entry.is_dir()
         )
```

Comment:
Make it a little easier to understand.

```suggestion
        # Sort by last modification
        entries = sorted(os.scandir(log_path), key=lambda entry: entry.stat().st_mtime)
        dirs.extend(entry.path for entry in entries if env in entry.name and entry.is_dir())
```